cicecore/cicedyn/analysis/ice_history.F90

```diff
@@ -2448,7 +2481,7 @@ subroutine accum_hist (dt)
 
          if (f_fswup(1:1) /= 'x') &
             call accum_hist_field(n_fswup, iblk, &
-                 (fsw(:,:,iblk)-fswabs(:,:,iblk)*workb(:,:)), a2D)
+                 (fsw(:,:,iblk)-fswabs(:,:,iblk)*aice(:,:,iblk)), a2D)
```

Comment:
In the original code, workb = aice_init.  Just making sure that this change to aice is intentional!

Reply:
Then the same bug fix also applied to flat, fsens, flowout, evap, fsurf and fcondtop?  That’s major and should also be noted in the PR header for incorporation into the merge commit.

Reply:
Duly noted.